.github/workflows/clean_dedicated_host.yml

```diff
@@ -0,0 +1,38 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT
+
+name: Clean Dedicated Host Daily Resources Cleaner
```

Comment:
Nitpick: I think the name is redundant. Personally, I think we can drop the "Clean" at the beginning of the name